stories/Table.stories.mdx

```diff
@@ -215,7 +228,7 @@ Use `size="sm"` to make tables compact by cutting cell padding in half.
 
 ## Sorting
 
```

Comment:
I think it will be good to mention what kind of sorting we are using in the docs so the users know how the sorting is done.  like javascript Array.sort() method is use , so they will know what to expect if they pass in an arrays of strings that has alphabets and number or maybe if there is symbols 

a use case i can think of is an array of id numbers

`["s942090", "s97302", "s89098236492"]`

Reply:
thanks @clukhei !